aerogear/unifiedpush-admin-client
# Aerogear 10278 - Added secret renew API

## Motivation
https://issues.redhat.com/browse/AEROGEAR-10278

## What
Added the `renewSecret` to the VariantAdmin class

## Why
The AdminUI has a feature to renew the secret

## Verification Steps
 
1. Open the UPS admin UI and create a variant
2. Take note of the variantid and the application id
3. Write a small program using this version of the client library and call the new api:
    ```javascript
    new UnifiedPushAdminClient('http://localhost:9999').
        variants.renewSecret('appId', 'variant type', 'variant id');
    ```
4. Refresh the AdminUI and verify that the variant secret has changed

## Checklist:
